ivy/container/linear_algebra.py

```diff
@@ -912,6 +942,17 @@ def matrix_norm(
         ret
             Matrix norm of the array at specified axes.
 
+        Examples
+        --------
+        >>> x = ivy.Container(a=ivy.array([[1.1, 2.2], [1., 2.]]), \
+                              b=ivy.array([[1., 2.], [3., 4.]]))
+        >>> y = x.matrix_norm(ord=1)
+        >>> print(y)
+        {
+            a: ivy.array(4.2),
+            b: ivy.array(6.)
+        }
```

Comment:
please also add examples to showcase different values of other args (`axis`, `keepdims`)